scripts/email_registration_contributors.py

```diff
@@ -0,0 +1,23 @@
+"""Script for sending OSF emails to all users who contribute to registrations."""
+
+from modularodm import Q
+
+from website import models
+from website.app import init_app
+from website.mails import Mail, send_mail
+
+
+def main():
+    registrations = models.Node.find(Q('is_registration', 'eq', True))
+    contributors = []
+    for node in registrations:
+        contributors.extend([contrib for contrib in node.contributors if contrib not in contributors])
```

Comment:
We need to make sure that users only receive this message once. Use the `security_messages` field on a User for this.